packer/http/preseed.cfg

```diff
@@ -0,0 +1,48 @@
+choose-mirror-bin mirror/http/proxy string
+d-i apt-setup/use_mirror boolean true
+d-i base-installer/kernel/override-image string linux-server
+d-i clock-setup/utc boolean true
+d-i clock-setup/utc-auto boolean true
+d-i finish-install/reboot_in_progress note
+d-i grub-installer/only_debian boolean true
+d-i grub-installer/with_other_os boolean true
+d-i keymap select us
+d-i mirror/country string manual
+d-i mirror/http/directory string /debian
+d-i mirror/http/hostname string httpredir.debian.org
+d-i mirror/http/proxy string
+d-i partman-auto-lvm/guided_size string max
+d-i partman-auto/choose_recipe select atomic
+d-i partman-auto/method string lvm
+d-i partman-lvm/confirm boolean true
+d-i partman-lvm/confirm boolean true
+d-i partman-lvm/confirm_nooverwrite boolean true
+d-i partman-lvm/device_remove_lvm boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+d-i partman/confirm_write_new_label boolean true
+d-i passwd/root-login boolean false
+d-i passwd/root-password-again password vagrant
+d-i passwd/root-password password vagrant
+d-i passwd/user-fullname string vagrant
+d-i passwd/user-uid string 1000
+d-i passwd/user-password password vagrant
+d-i passwd/user-password-again password vagrant
+d-i passwd/username string vagrant
+d-i pkgsel/include string sudo wget curl make openssh-server
```

Comment:
My vision for preseed.cfg was that we only install the barest minimum tools we needed to get the thing to build. There are other tools at https://github.com/chef/bento/blob/master/debian/http/debian-8/preseed.cfg#L33 that I removed for that reason.

Upon review, I think we should add `acpid` after sudo because I think it's needed to handle power on/off signals in the hypervisor. See https://unix.stackexchange.com/questions/73877/do-i-need-acpid-in-a-virtual-machine